src/test/scala/com/microsoft/hyperspace/index/dataskipping/DataSkippingIndexTest.scala

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (2021) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.microsoft.hyperspace.index.dataskipping
+
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.Path
+import org.apache.spark.sql.SaveMode
+import org.apache.spark.sql.functions.{input_file_name, max, min}
+
+import com.microsoft.hyperspace.index.{FileInfo, Index}
+import com.microsoft.hyperspace.index.dataskipping.sketch.{MinMaxSketch, ValueListSketch}
+
+class DataSkippingIndexTest extends DataSkippingSuite {
+  test("kind") {
```

Comment:
Can we have a better naming convention? 
https://www.puresourcecode.com/dotnet/testing/7-popular-unit-test-naming-conventions/

Reply:
Done